distribution/package.xml

```diff
@@ -10,24 +10,12 @@
       <filter token="current.year" value="${current.year}" />
    </filterset>
 
-   <target name="package" depends="clean, wf-modules" />
+   <target name="package" depends="clean" />
```

Comment:
can you try to remove this file and invocation from pom.xml? the only target is `clean` at this point :shrug: